tests/cl-go-client/main.go

```diff
@@ -561,6 +572,106 @@ func createPoolOp(igniteClient cosmosclient.Client) {
 	}
 }
 
+func claimSpreadRewardsOp(igniteClient cosmosclient.Client) {
+	var (
+		randAccountNum = rand.Intn(8) + 1
+		accountName    = fmt.Sprintf("%s%d", accountNamePrefix, randAccountNum)
+	)
+
+	// Instantiate a query client
+	clClient := clqueryproto.NewQueryClient(igniteClient.Context())
+
+	accountMutex.Lock() // Lock access to getAccountAddressFromKeyring
+	senderAddress := getAccountAddressFromKeyring(igniteClient, accountName)
+	accountMutex.Unlock() // Unlock access to getAccountAddressFromKeyring
+
+	userPositionResp, err := clClient.UserPositions(context.Background(), &clqueryproto.UserPositionsRequest{
+		Address: senderAddress,
+		PoolId:  0,
```

Comment:
Is this default poolId that always contain positions?

Reply:
this lists all positions instead of just a specific pool